tests/slow/src/test/scala/tests/feature/CompletionCrossLspSuite.scala

```diff
@@ -154,6 +154,151 @@ class CompletionCrossLspSuite
     } yield ()
   }
 
+  test("implicit-class-circe-dependency") {
```

Comment:
If I understand the unit tests correctly, this PR covers code completion and auto-import of:
- implicit methods from the workspace
- implicit methods from the external dependencies
- implicit methods from the Scala standard library

This is great! 👍 It was mentioned that it builds upon the #7791, which was said to support the top-level Scala 3 extension methods as well. Do these changes cover the extensions too, or is it left for a future follow up? 🙂

Reply:
Thanks, this is already a big and very welcome improvement, but nice to see that further extensions (pun intended 😉) are still coming. 👍